metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/rest/execution/details/PluginProgressView.java

```diff
@@ -0,0 +1,41 @@
+package eu.europeana.metis.core.rest.execution.details;
+
+import eu.europeana.cloud.common.model.dps.TaskState;
+import eu.europeana.metis.core.workflow.plugins.ExecutionProgress;
+
+public class PluginProgressView {
```

Comment:
Add javadoc

Reply:
I guess I forgot to add Javadoc in those new files. I'll add it.